build-server/src/main/resources/dockerfiles/java-maven/Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM ubuntu:precise
 
 MAINTAINER Michael de Jong "https://github.com/michaeldejong"
 
-RUN echo "deb http://archive.ubuntu.com/ubuntu precise main universe" > /etc/apt/sources.list
```

Comment:
Needed to change this to get the Dockerfile to build on my Docker version (1.4.0)